app/api/dao/organization.py

```diff
@@ -0,0 +1,104 @@
+
+from http import HTTPStatus
+from flask import json
+from app.database.models.bit_schema.organization import OrganizationModel
+from app.database.models.bit_schema.user_extension import UserExtensionModel
+from app import messages
+from app.api.request_api_utils import AUTH_COOKIE
+from app.utils.bitschema_utils import Timezone, OrganizationStatus
+
+
+class OrganizationDAO:
+    
+    """Data Access Object for Users_Extension functionalities"""
+
+    @staticmethod
+    def get_organization(rep_id):
+        """Retrieves the organization that is represented by the user which ID is passed as parameter.
+
+        Arguments:
+            rep_id: The ID of the user who represents the organization.
+
+        Returns:
+            The OrganizationModel class represented by the user whose ID was searched.
+        """
+
+        rep_additional_info = UserExtensionModel.find_by_user_id(rep_id)
```

Comment:
Maya it's a good thing to not use abbreviations. It gets really difficult when you're code is worked on by more people. Try not using abbreviations anywhere, could be `representative_id`.

Reply:
ok, will change it.